indico/modules/events/editing/templates/emails/base_email.txt

```diff
@@ -0,0 +1,8 @@
+{% extends 'emails/base.txt' %}
+
+{% block header_recipient -%} {{ recipient_name }} {%- endblock %}
```

Comment:
this space here and in the other templates is a bit weird.. since it gets stripped anyway I'd just use linebreaks (and then indentation as well) if you want to keep some space for readability:

```
{% block header_recipient -%}
    {{ recipient_name }}
{%- endblock %}
```

or

```
{% block header_recipient -%}{{ recipient_name }}{%- endblock %}
```